ocs_ci/ocs/platform_nodes.py

```
@@ -3874,3 +3878,293 @@ def create_nodes(self, node_conf, node_type, num_nodes):
+            max_retries = 5
+            for attempt in range(max_retries):
+                try:
+                    logger.info(
+                        f"Attempting to verify node status via API (attempt {attempt + 1}/{max_retries})..."
+                    )
+
+                    # Try to re-login if we lost authentication
+                    try:
+                        refresh_oc_login_connection()
+                        logger.info("Successfully re-authenticated to cluster")
+                    except Exception as login_error:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)